x-pack/plugins/ml/public/application/jobs/jobs_list/components/job_details/forecasts_table/forecasts_table.js

```diff
@@ -302,48 +371,73 @@ export class ForecastsTable extends Component {
         textOnly: true,
       },
       {
-        name: i18n.translate('xpack.ml.jobsList.jobDetails.forecastsTable.viewLabel', {
-          defaultMessage: 'View',
+        width: '75px',
+        name: i18n.translate('xpack.ml.jobsList.jobDetails.forecastsTable.actionsLabel', {
+          defaultMessage: 'Actions',
         }),
-        width: '60px',
-        render: (forecast) => {
-          const viewForecastAriaLabel = i18n.translate(
-            'xpack.ml.jobsList.jobDetails.forecastsTable.viewAriaLabel',
-            {
-              defaultMessage: 'View forecast created at {createdDate}',
-              values: {
-                createdDate: timeFormatter(forecast.forecast_create_timestamp),
-              },
-            }
-          );
-
-          return (
-            <EuiButtonIcon
-              onClick={() => this.openSingleMetricView(forecast)}
-              isDisabled={
+        actions: [
+          {
+            description: i18n.translate('xpack.ml.jobsList.jobDetails.forecastsTable.viewLabel', {
+              defaultMessage: 'View',
+            }),
+            type: 'icon',
+            icon: 'singleMetricViewer',
```

Comment:
As discussed on slack, can we change this one to the EUI:eye?

Reply:
```suggestion
            icon: 'eye',
```

Reply:
Sorry about that - updated in 79038f0a53e062dacf1a65e8f561c5f72a84ea18